daemon/config.go

```diff
@@ -85,6 +84,7 @@ type CommonConfig struct {
 	// deserialization without the full struct.
 	CommonTLSOptions
 	LogConfig
+	bridgeConfig // bridgeConfig holds bridge network specific configuration.
```

Comment:
i wonder if it's gonna get marshalled, because even though it is embedded, it is not exported. Will check.